tsconfig.json

```diff
@@ -18,10 +18,14 @@
     "**/*.test.js",
     "**/*.test.ts",
     "**/*.test.tsx",
+    "**/jest-coverage/**/*",
+    "**/__mocks__/**/*",
+    "**/storybook-build/**/*",
     ".storybook/**/*",
     "builds/**/*",
     "dist/**/*",
-    "node_modules/**"
+    "node_modules/**",
+    "development/ts-migration-dashboard/**/*"
```

Comment:
There are TypeScript files inside of the dashboard code. Assuming that the goal is to exclude output files, should this be `development/ts-migration-dashboard/build/**/*` and `development/ts-migration-dashboard/intermediate/**/*`?

Reply:
Done :)